src/Lean/Server/Requests.lean

```diff
@@ -38,7 +38,7 @@ def methodNotFound (method : String) : RequestError :=
     message := s!"No request handler found for '{method}'" }
 
 instance : Coe IO.Error RequestError where
-  coe e := { code := ErrorCode.internalError
+  coe e := { code := ErrorCode.workerCrashed
```

Comment:
This is a bit non-obvious, but the error constructed here should *also* only be emitted without the worker crashing. It is returned from `RequestM` calls that fail for some `IO` reason like a missing file. The worker should be able to handle those failure graciously and without crashing (if it doesn't, that's a bug). So this should also be `internalError`.